modules/monitoring-support-version-matrix-for-monitoring-components.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="support-version-matrix-for-monitoring-components_{context}"]
+= Support version matrix for monitoring components
+
+The following matrix contains information about versions of monitoring components for {product-title} 4.12 and later releases:
+
+|===
+|*{product-title}* 11+|*Component Versions*
+
+|*Version* |*Prometheus Operator* |*Prometheus*  |*Prometheus Adapter* |*Metrics Server* (Technology Preview) |*Alertmanager* |*kube-state-metrics agent* |*monitoring-plugin* |*openshift-state-metrics agent* |*node-exporter agent* |*Thanos* |*Telemeter Client*
+
+|4.16 |0.73.2 |2.51.2 |0.11.2 |0.7.1 |0.26.0 |2.12.0 |1.0.0 |TBD |1.8.0 |0.35.0 |TBD 
+
+|4.15 |0.70.0 |2.48.0 |0.11.2 |0.6.4 |0.26.0 |2.10.1 |1.0.0 |TBD |1.7.0 |0.32.5 | TBD 
+
+|4.14 |0.67.1 |2.46.0 |0.10.0 |0.6.4 |0.25.0 |2.9.2 |1.0.0 |TBD |1.6.1 |0.30.2 | TBD 
+
+|4.13 |0.63.0 |2.42.0 |0.10.0 |N/A |0.25.0 |2.8.1 |N/A |TBD |1.5.0 |0.30.2 | TBD 
+
+|4.12 |0.60.1 |2.39.1 |0.10.0 |N/A |0.24.0 |2.6.0 |N/A |TBD |1.4.0 |0.28.1 | TBD 
+|===
```

Comment:
Hello @juzhao I could not find any information about the versions of openshift-state-metrics agent and Telemeter Client.  Could you help me with this? Thank you!

Reply:
(Also, in order to CP this to older versions, if it is possible, I need the versions for these 2 components from 4.10+ OCP releases.)

Reply:
openshift-state-metrics and telemeter client are OCP-specific components so their versions would be the same as the OCP version (e.g. openshift-state-metrics version for OCP 4.16 is 4.16 too).

Reply:
for openshift-state-metrics and telemeter-client version, we have not mention them in our official docs.
maybe @simonpasquier know the reaseon
and only can know below release version for openshift-state-metrics, example
```
$ oc get clusterversion
NAME      VERSION                              AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.12.0-0.nightly-2024-05-11-022650   True        False         11h     Cluster version is 4.12.0-0.nightly-2024-05-11-022650

$ oc -n openshift-monitoring exec -c openshift-state-metrics openshift-state-metrics-5495f66cbc-cwqwb -- openshift-state-metrics --version
version.Version{GitCommit:"74fec74", BuildDate:"2024-05-09T20:15:16Z", Release:"v4.12.0-202405091536.p0.g3d5dc18.assembly.stream.el8", GoVersion:"go1.19.13 X:strictfipsruntime", Compiler:"gc", Platform:"linux/amd64"}
```
no --version flag for telemeter-client, don't know the version either.

Reply:
Oh that explains it, thank you very much for the information!